addon/index.js

```diff
@@ -146,6 +158,10 @@ export class EmberChangeset extends BufferedChangeset {
     // 'person.username'
     let [baseKey, ...remaining] = key.split('.');
 
+    if (NOPROXYPROPS.includes(key)) {
+      return this[key];
+    }
+
```

Comment:
What do you think about this below on line 190?

```js
if (typeof this[key] !== 'undefined') {
```

Reply:
There is a lot of complication here and off the top of my head wouldn't the type of the property `isValid` return [boolean](https://github.com/validated-changeset/validated-changeset/blob/master/src/index.ts#L209)?

Reply:
Yep and that is the goal.  The simple fix here is to change below from.

```js
if (this[key]) {
```

```js
if (typeof this[key] !== 'undefined') {
```

`isValid` and other properties that also exist on an Ember Data model won't be reflected via accessing through a changeset.    If the user asks the question `isValid`, the answer they should get back is if the changeset is valid.

Reply:
I updated my pull request and rebased.